src/include/netloc_util.h

```diff
@@ -0,0 +1,52 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/*
+ *  (C) 2001 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ *
+ */
+
+#ifndef NETLOC_UTIL_H_INCLUDED
+#define NETLOC_UTIL_H_INCLUDED
+
+#include "netloc.h"
+
+typedef enum {
+    MPIR_NETLOC_NETWORK_TYPE__FAT_TREE,
+    MPIR_NETLOC_NETWORK_TYPE__CLOS_NETWORK,
+    MPIR_NETLOC_NETWORK_TYPE__INVALID,
+} MPIR_Netloc_network_topo_type;
+
+typedef struct {
+    netloc_node_t *node;
+    long index;
+} node_index;
```

Comment:
I still see `node_index`.  Please really fix my comments.

Reply:
I am so sorry about that, I have fixed this.